Tests/scripts/update_id_set.py

```diff
@@ -396,6 +472,73 @@ def re_create_id_set():
     exec_time = time.time() - start_time
     print_color("Finished the creation of the id_set. Total time: {} seconds".format(exec_time), LOG_COLORS.GREEN)
 
+    duplicates = find_duplicates(new_ids_dict)
```

Comment:
performance wise - no need to run this every build, should be removed to validate_files in the future (and verified only on relevant IDs and not all).

Reply:
I'm facing the issue of which files to test in secrets too, to be honest I don't think it should happen inside the scripts but only in the file validator. With a circle/no circle distinction